### ISSUE TITLE
[Enhancement] More CSS samples

### ISSUE THREAD

**Issue by Depechie** [xamarin/xamarin-forms-samples#457]
### Sample

https://github.com/xamarin/xamarin-forms-samples/tree/master/UserInterface/Styles/MonkeyAppCSS

### Description

Could the sample be expanded to show more CSS capabilities that should be available following the documentation?

### Information

If I read the docs about CSS correctly, Span should be targetable through CSS for text-decoration.
But I'm unable to get this working inside a Label > FormattedText > Span.
Could be I'm doing something wrong, so was looking for a sample.

Would be nice if the CSS sample was expanded to show examples of more targets, following the docs.

**Comment by profexorgeek** [comment]
We have added a work item to investigate and improve our CSS docs. The work item references this issue but will likely be wider in scope.

**Comment by Depechie** [comment]
@profexorgeek awesome news!

**Comment by profexorgeek** [comment]
Hi @Depechie,
I spent some time digging into this issue this week. I was able to repro your issue, which has a triaged bug (filed by you) open against it: https://github.com/xamarin/Xamarin.Forms/issues/5979

I made a small update to the doc to note the bug. Beyond that I don't think it makes sense to add to the CSS docs/sample: they covers all of the common use cases for CSS styling accurately. If `Span` functionality wasn't broken, your attempts would have worked as you expected!
